frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/PageAccordion.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { ReactNode } from 'react';
 import classes from './PageAccordion.module.css';
+import cn from 'classnames';
 import { Accordion } from '@digdir/design-system-react';
 import { NavigationMenu } from './NavigationMenu';
+import * as testids from '../../../../../../testing/testids';
```

Comment:
Dette er utenfor scope av PR, men jeg er fristet til å flytte testids ut til et annet sted, for å unngå å dra inn avhengigheter fra testing mappen i produksjonskoden vår. 😊

Reply:
Ja, kan sikkert være lurt å refaktorere det i en egen PR :)